tests/basic/misc_test.go

```diff
@@ -1784,4 +1784,71 @@ var _ = Describe("{KubeClusterRestart}", func() {
 		defer EndTorpedoTest()
 		AfterEachTest(contexts, testrailID, runID)
 	})
-})
\ No newline at end of file
+})
+
+var _ = Describe("{VerifyNoPxRestartDueToPxPodStop}", func() {
+	JustBeforeEach(func() {
+		var testrailID = 87311294
+		StartTorpedoTest("VerifyNoPxRestartDueToPxPodStop", "Verify that px serivce remain up even if px pod got deleted ", nil, 0)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	It("has to setup, validate and teardown apps", func() {
+
+		// Get uptime for px service on each node
+		stepLog = "Getting PID of Px process before and after PX pods on all the nodes"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			processPid := make(map[string]string)
+			processCmd := "pidof px"
+			storageNode := node.GetStorageNodes()
+			storageLessNodes := node.GetStorageLessNodes()
+			var AllNodes []node.Node
+			AllNodes = append(storageNode, storageLessNodes...)
+			//Capturing PID pf PX before stopping PX pods
+			for _, nnode := range AllNodes {
+				output, _ := Inst().N.RunCommand(nnode, processCmd, node.ConnectionOpts{
```

Comment:
We might need to handle the error returned by `Inst().N.RunCommand`.

Reply:
Done